core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -188,6 +221,42 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
       });
   }
 
+  onGroupDelete(rows: Set<WorkflowExecutionsEntry>) {
+    const modalRef = this.modalService.open(DeletePromptComponent);
+    modalRef.componentInstance.deletionType = "execution";
+    let deletionNames = "";
+    for (let row of rows) {
```

Comment:
why is this needed?

Reply:
I need this for loop to update bookmark locally

Reply:
I believe this is for deletion not bookmark. Please avoid unnecessary steps